docs/experiment_tracking.md

```diff
@@ -0,0 +1,88 @@
+# Experiment tracking in llama recipes. 
+
+Llama recipes has an optional support for tracking experiments using experiment tracker modules like Aimstack or WandB (Currently not supported).
```

Comment:
nit:
I'd drop 'an' in 'an optional'

I think "Currently only Aimstack is supported" might be clearer, "not supported' doesn't make it clear what is/isn't.